usaspending_api/references/models.py

```diff
@@ -247,7 +250,7 @@ class LegalEntity(DataSourceTrackedModel):
     class Meta:
         managed = True
         db_table = 'legal_entity'
-        unique_together = (('recipient_name'),)
+        unique_together = (('recipient_unique_id'),)
```

Comment:
I wanted to leave it open to have combination of fields determine uniqueness, namely `recipient_unique_id` and some form of date